frontend/src/components/limited-textarea/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Text, Textarea, TextareaProps } from "theme-ui";
+
+export const LimitedTextarea = ({
+  value,
+  maxLength,
+  ...props
+}: TextareaProps) => {
+  return (
+    <>
+      <Textarea value={value} maxLength={maxLength} {...props} />
+      <Text
+        variant="labelDescription"
+        as="p"
+        mb={4}
+        color={(value as string)?.length >= maxLength ? "red" : "black"}
+      >
+        {(value as string)?.length}/{maxLength}
```

Comment:
What types can `value` be? also using `value?.` doesn't seem ideal since this will throw an error because it will end up being `undefined.length`?

Reply:
can we maybe change the value type so it is always a string?

Reply:
Typescript was red because it can be null | string | string[]

Reply:
can you check again? I set value to string now :)

Reply:
No typescript complains again... 
> Error: src/components/limited-textarea/index.tsx(13,8): error TS2322: Type '{ ref?: LegacyRef<HTMLTextAreaElement>; key?: Key; autoComplete?: string; autoFocus?: boolean; cols?: number; dirName?: string; disabled?: boolean; form?: string; ... 296 more ...; maxLength: number; }' is not assignable to type 'RefAttributes<HTMLTextAreaElement>'.

But I don't know how to solve this... :(